web3rpc/sdk/client/javascript/template/libraries/javascript/api.mustache

```diff
@@ -57,12 +67,16 @@ export default class <&classname> {
 </allParams>
 
       let postBody = <#bodyParam>new <dataType>(undefined, undefined, undefined)</bodyParam>;
+      let optsIndex = 0;
       postBody.params = [];
       <#queryParams><#required>
       postBody.params.push(<paramName>);</required><^required>
-      if(opts['<paramName>']) {
-        postBody.params.push(opts['<paramName>']);
-      }</required></queryParams>
+      var optParam = opts['<paramName>'] || opts[optsIndex++];
+      if (optParam === undefined || optParam === null) {
```

Comment:
It's intended for users necessarily to enter the opts?

Reply:
Yes, it is intended. Currently, there seems no '<defualtValue>' passed to the template, so the user must supply them. If I don't throw it here, then the user gets a server RPC error which is less comprehensible.

We might fix 'codegen/src/main/kotlin/web3rpc/client/KlaytnJavascriptClientCodegen.kt' like Java did, but I did not.

Reply:
@kjeom Fixed the Kotlin code. PTAL.

Reply:
@blukat29 LGTM